components/AvailableModal.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, ActivityIndicator, TouchableOpacity} from 'react-native';
+import PrimaryButton from './PrimaryButton';
+
+
+export default class AvailableModal extends React.Component {
+  constructor(props){
+    super(props);
+      this.state = {items: [this.props.name, this.props.packageSize, this.props.location]};
+  }
+
+  static navigationOptions = {
+    header: null,
+    };
+
+
```

Comment:
Where's the `accept` function? You should implement it so the modal can close when the user presses the button